com.unity.shadergraph/Editor/Resources/Styles/Controls/EnumControlView.uss

```diff
@@ -16,9 +16,10 @@ EnumControlView {
 }
 
 EnumControlView > Label {
-    max-width: 94px;
+    max-width: 140px;
```

Comment:
Does this still look good on the other enum controls on other nodes?

Reply:
I looked at all of the nodes in the library both before and after this change. Although there are several nodes that have crowded or overlapping drop-down descriptions (UV, Scene Depth, Gradient Noise, Simple Noise) none of them appear to have changed based on this.  They're the same both with and without it.

Reply:
Thank you, Ben! I also poped some nodes that have enum controls and they all look fine.
![image](https://user-images.githubusercontent.com/67800322/136270379-a8353328-3836-4ca2-bc3c-fca37675f509.png)